### ISSUE TITLE
float division by zero

### ISSUE THREAD

**Issue by peter91011** [erikbern/deep-pink#21]
Hi, I got an error when running train.py
I have already download pgn files in the working directory.
When creating function get_data:

test_size = 10000.0 / len(data[0])
print('Splitting', len(data[0]), 'entries into train/test set')
data = train_test_split(*data, test_size=test_size)

len(data[0])  is 0. So comes to the error.
Do you know how to fix that?
I'm sure I have already downloaded pgn files and run parse_game.py at first

**Comment by erikbern** [comment]
no idea, haven't looked at this code for several years!

**Comment by XOGautham** [comment]
Did you find anything

**Comment by mochong** [comment]
I also meet this problem

**Comment by erikbern** [comment]
this code is extremely old, probably doesn't make sense to try to run it
